### ISSUE TITLE
It seems some problems in the implementation of JMMD (Deep Transfer Learning with Joint Adaptation Networks)

### ISSUE THREAD

**Issue by less-and-less-bugs** [thuml/Transfer-Learning-Library#158]
Hi, the jmmd loss will be negative given the existing implementation of JMMD in class JointMultipleKernelMaximumMeanDiscrepancy(nn.Module).

I think may be another approximation of JMMD should be better.

**Comment by less-and-less-bugs** [comment]
Thank you very much.
